dts/msm8916/msm8939-xiaomi-ido.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <0xef 0x30000 0xf1 0x30000 0x107 0x30000>;
```

Comment:
Consider changing turning this into:
`qcom,msm-id = <239 0x30000>, <241 0x30000>, <263 0x30000>;`